source/_posts/2018-01-12-pager-duty-and-confluence.markdown

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: "Pager Duty and Confluence"
+date: 2018-01-12 18:19:49 +0000
+comments: true
+author: "Bryan Kroger bryan.kroger@grandrounds.com"
+categories: automation
+---
+My first project here at Grand Rounds was to pick up one of my favorite topics of IT and operations 
+in general: on-call rotations!  Making sure we have accurate alarming and alerting never seems like the
+most glamours work in the world, but it is often very important and highly impactful.
+
+To that end, as with most things, we want to start with a data-driven approach to identifying and resolving
+our problems.  This is a pattern that I've used several times over the years for a fun little hack which
+allows us to build reports based on PagerDuty incident data.
+
+This gives us two graphs:
+
+* Number of incidents per service.
+* Amount of time each service spends in incident ( on average ).
+
+We fully expect that this code is a little rough around the edges, and that's okay.  We're building things
+here that are going to give us ideas and insight into our bigger picture items.  Ideally we want to
+spend as little time as possible gold plating this code because we're not 100% sure this will be something
+that we intend to live forever.  Just like in the startup business world we want to create the Minimal
+Viable Product ( MVP ) which allows us to prove out a business case.
+
+In this case we will prove out the value of incident reporting data.  This is a perfect example of a product
+that is valuable today, but becomes less valuable over time because it allows us to refine a process.
+
+We fully expect that some day this data will be effictively useless and thus this code will be thrown
+away.  However, the value of this code today is in it's ability to give us insights that we can act on
+today.
```

Comment:
nit: I feel like this paragraph duplicates the content of the previous two paragraphs, feels repetitive.